setup.py

```diff
@@ -18,6 +18,16 @@ def dependencies(file):
     packages=find_packages(exclude=("tests", "examples")),
     version="0.0.30",
```

Comment:
Can you increment the version to make a release?

Reply:
Sure thing, done!